adafruit_platformdetect/board.py

```diff
@@ -343,6 +352,8 @@ def id(self):
             board_id = MICROCHIP_MCP2221
         elif chip_id == ap_chip.BINHO:
             board_id = BINHO_NOVA
+        elif chip_id == ap_chip.ONION:

```

Comment:
Looks like we need to change this line to
`elif chip_id == ap_chip. MIPS24KC:`

If it says build failed, you can click the Checks tab and then click on the "The Build" link to see why.

Reply:
Never mind, you're quick. :)

Reply:
I noticed that when I made the push, I have fixed it already. :yum: